brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerResponseAdapter.java

```diff
@@ -18,12 +18,8 @@ public HttpServerResponseAdapter(HttpResponse response)
 
     @Override
     public Collection<KeyValueAnnotation> responseAnnotations() {
-        int httpStatus = response.getHttpStatusCode();
-
-        if ((httpStatus < 200) || (httpStatus > 299)) {
-            KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create("http.responsecode", String.valueOf(httpStatus));
-            return Arrays.asList(statusAnnotation);
-        }
-        return Collections.EMPTY_LIST;
+        KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create(
```

Comment:
I don't think it will be a significant change in size to log status code... size is a concern, but I'd rather have fewer traces with more data in each, than more traces with less data in each, does that make sense?

One possible long-term optimization could be to have "debug" annotations that are only logged when you trigger a trace in debug mode (e.g. with the browser extension)